shelley/design-spec/delegation_design_spec.tex

```diff
@@ -796,33 +796,6 @@ \subsubsection{Bootstrap Address}
 have essentially identical behaviour. This might change in the future, if new
 features are added to enterprise addresses.
 
-\subsubsection{Script Address}
```

Comment:
Or alternatively we could describe these things as script addresses rather than multisig addresses. I think it'd be clearer and show the direction of travel, and be easier to extend when we add Plutus Core, lightning etc. The requirement is for multisig, the design is for script addresses, and the Shelley release does mutlisig via script addresses with a small specialised multisig language. I think that's not too hard a story to tell, and means less to change later as we extend things.

Reply:
Do you think my recent reformulation of the "Address Structure" section does that well?